### PR TITLE
Fixes to links in README.md

### PR DESCRIPTION
**Pull request by SirRedrick**

### CODE REVIEW

**Review by luamoris — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 ## Team
 
-- Дмитрий Гришаев - [Telegram](t.me/iDisbalance)
-- Игорь Мельник - [Telegram](t.me/solev0i)
-- Никита Батрак - [Telegram](t.me/Sir_Redrick)
-- Иосиф Исачко - [Telegram](t.me/I0S1F)
```
Comment (luamoris) [inline README.md:8]: The problem with names

File: README.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
+- Дмитрий Гришаев - [Telegram](https://www.t.me/iDisbalance)
+- Игорь Мельник - [Telegram](https://www.t.me/solev0i)
+- Никита Батрак - [Telegram](https://www.t.me/Sir_Redrick)
+- Игорь Мельник - [Telegram](https://www.t.me/I0S1F)
```
Comment (luamoris) [inline README.md:8]: The problem with names.

**Review by luamoris — CHANGES_REQUESTED** [review]
``

**Review by floumaster — COMMENTED** [review]
So fucking good

**Review by luamoris — APPROVED** [review]
Good!

**Review by Solevo1 — APPROVED** [review]
Nuts.bet

**Review by floumaster — APPROVED** [review]
Ф5

**Pull request merged**